core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
This works for Params, Cookies, and Headers but we're seeing that this doesn't work for a Request Body readers. Required Bodies are actually just `ConstReader`s. I think it would be wrong to make `ConstReader`s report it was a request body and I'm not sure we want to replicate the logic of `ConstReader` inside the Required Bodies.